install_config/aggregate_logging.adoc

```diff
@@ -2344,47 +2344,28 @@ $ curl --key /etc/elasticsearch/secret/admin-key \
 [[fluentd-redeploy-certs]]
 == Redeploying EFK Certificates
 
-You can redeploy EFK certificates, if needed.
+You can use an Ansible playbook to perform a certificate rotation for the EFK stack without needing to run the install/upgrade playbook.
+
+This playbook deletes the current certificate files, generates new EFK certificates, updates certificate secrets, and restarts components 
```

Comment:
so, the playbook doesn't restart every component -- just kibana and elasticsearch. Not sure if we want to point that out.

fluentd will reload the certificates upon failing to connect to elasticsearch and a curator pod is created based on the cronjob schedule.

Reply:
Thanks for the clarification. I took the DocText in th eBZ too literally, it appears!